src/Sylius/Behat/Context/Api/Admin/ManagingCustomersContext.php

```diff
@@ -0,0 +1,590 @@
+    }
+
+    /**
+     * @Then I should be notified that email must be unique
+     */
+    public function iShouldBeNotifiedThatEmailMustBeUnique(): void
+    {
+        Assert::contains(
+            $this->responseChecker->getError($this->client->getLastResponse()),
+            'email: This email is already used.',
+        );
+    }
+
+    /**
+     * @Then /^I should be notified that ([^"]+) should be ([^"]+)$/
+     */
+    public function iShouldBeNotifiedThatTheElementShouldBe($elementName, $validationMessage): void
+    {
+        Assert::contains(
+            $this->responseChecker->getError($this->client->getLastResponse()),
+            sprintf('%s must be %s.', ucfirst($elementName), $validationMessage),
+        );
+    }
+
+    /**
+     * @Then I should be notified that email is not valid
+     */
+    public function iShouldBeNotifiedThatEmailIsNotValid(): void
+    {
+        Assert::contains(
+            $this->responseChecker->getError($this->client->getLastResponse()),
+            'This email is invalid.',
+        );
+    }
+
+    /**
+     * @Then the customer :customer should appear in the store
+     * @Then the customer :customer should still have this email
+     */
+    public function theCustomerShouldAppearInTheStore(CustomerInterface $customer): void
+    {
+        Assert::true(
+            $this->responseChecker->hasItemWithValue($this->client->index(Resources::CUSTOMERS), 'email', $customer->getEmail()),
+            sprintf('Customer with email %s does not exist', $customer->getEmail()),
+        );
+    }
+
+    /**
+     * @Then the customer :customer should have an account created
+     * @Then /^(this customer) should have an account created$/
+     */
+    public function theyShouldHaveAnAccountCreated(CustomerInterface $customer): void
+    {
+        Assert::notNull(
+            $customer->getUser()->getPassword(),
+            'Customer should have an account, but they do not.',
+        );
+    }
+
+    /**
+     * @Then I should see :count customers in the list
+     * @Then I should see a single customer on the list
+     */
+    public function iShouldSeeZonesInTheList(int $count = 1): void
+    {
+        Assert::same($this->responseChecker->countCollectionItems($this->client->getLastResponse()), $count);
+    }
+
+    /**
+     * @Then I should see the customer :email in the list
+     */
+    public function iShouldSeeTheCustomerInTheList(string $email): void
+    {
+        Assert::true(
+            $this->responseChecker->hasItemWithValue($this->client->index(Resources::CUSTOMERS), 'email', $email),
+            sprintf('There is no customer with email "%s"', $email),
+        );
+    }
+
+    /**
+     * @Then I should see a single order in the list
+     */
+    public function iShouldSeeASingleOrderInTheList(): void
+    {
+        Assert::same($this->responseChecker->countCollectionItems($this->client->getLastResponse()), 1);
+    }
+
+    /**
+     * @Then his name should be :name
```

Comment:
We should probably slightly refactor scenarios to use `their` instead of `his`/`her`. The same applies to the below steps